src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,362 @@
+                image: newrelic/infrastructure-k8s:2.4.0-unprivileged
+                imagePullPolicy: IfNotPresent
+                resources:
+                  limits:
+                    memory: 300M
+                  requests:
+                    cpu: 100m
+                    memory: 150M
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  readOnlyRootFilesystem: true
+                  runAsUser: 1000
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+                volumeMounts:
+                - mountPath: /var/db/newrelic-infra/data
+                  name: tmpfs-data
+                - mountPath: /var/db/newrelic-infra/user_data
+                  name: tmpfs-user-data
+                - mountPath: /tmp
+                  name: tmpfs-tmp
+                - mountPath: /var/cache/nr-kubernetes
+                  name: tmpfs-cache
+        [...]
+        ```
+      </Collapser>
+    </CollapserGroup>
+
+3. Create a `ClusterRoleBinding`, or add to a previously created one the `ServiceAccount` of the application that is going to be monitored. All the workloads may share the same `ClusterRoleBinding`, but the `ServiceAccount` of each one must be added to it.
+
+    <CollapserGroup>
+      <Collapser
+        id="cluster-role-binding"
+        title="clusterRoleBinding.yaml"
+      >
+      [PLEASE INSERT INSTRUCTIONS ABOUT HOW TO USE THIS SNIPPET. ALSO, I NOTICED IN THE PRIVATE REPO THAT THE FIRST LETTER OF THE YAML FILE (ClusterRoleBinding.yaml) IS CAPITALIZED, UNLIKE OTHER YAML FILE NAMES. IT LOOKS LIKE THE OBJECTS INSIDE THE FILES START WITH CAPITAL LETTERS WHILE THE YAML FILES DO NOT, RIGHT?]
+
+      ``` 
+      apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: newrelic-newrelic-infrastructure-infra-agent
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: newrelic-newrelic-infrastructure-infra-agent
+      subjects:
+      - kind: ServiceAccount
+        name: [<var>INSERT_SERVICE_ACCOUNT_NAME_OF_WORKLOAD</var>]
+        namespace: [<var>INSERT_SERVICE_ACCOUNT_NAMESPACE_OF_WORKLOAD</var>]
+      ```
+      </Collapser>
+    </CollapserGroup>
+
+
+4. Create a secret containing the New Relic license key. Each namespace needs its own secret.
+
+    <CollapserGroup>
+      <Collapser
+        id="secret"
+        title="secret.yaml"
+      >
+      [PLEASE INSERT INSTRUCTIONS ABOUT HOW TO USE THIS SNIPPET]
+
+      ```      
+      apiVersion: v1
+      data:
+        license: superSecretLicenseEncodedInBase64===
```

Comment:
also this can be a var, so that they keep in mind to change that

Reply:
@paologallinaharbur, I added the <var> tags, but I also changed the format so it matches other user-input fields. I hope that trailing "===" was not specific to Base64.

license: <var>INSERT_YOUR_NEW_RELIC_LICENSE_ENCODED_IN_BASE64</var>